gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePlugin.java

```diff
@@ -276,6 +299,22 @@ private static List<String> checksToApplySuggestedPatchesFor(
                 .collect(Collectors.toList());
     }
 
+    private static List<String> checksToRemoveSuppressionsFor(JavaCompile javaCompile) {
```

Comment:
I think this can take `Project` instead of `JavaCompile`?

Reply:
Indeed - didn't catch that javaCompile.getProject was essentially just the project 🤦